crates/bevy_math/src/isometry.rs

```diff
@@ -206,7 +268,81 @@ impl UlpsEq for Isometry2d {
     }
 }
 
-/// An isometry in three dimensions.
+/// An isometry in three dimensions, representing a rotation followed by a translation.
+/// This can often be useful for expressing relative positions and transformations from one position to another.
+///
+/// In particular, this type represents a distance-preserving transformation known as a *rigid motion* or a *direct motion*,
+/// and belongs to the *Special Euclidean group* SE(3). This includes translation and rotation, but excludes reflection.
```

Comment:
Wikipedia or similar links would be nice for further reading.

Reply:
Seconding this!